lib/cartopy/crs.py

```diff
@@ -1250,7 +1250,7 @@ def quick_vertices_transform(self, vertices, src_crs):
         if vertices.size == 0:
             return vertices
 
-        if self == src_crs:
+        if self == src_crs and len(vertices) > 0:
```

Comment:
Because of the lines 1250-1251, which were added at #2213, I don’t think this check will ever be exercised.